Assets/Scripts/Game/Entities/EnemyEntity.cs

```diff
@@ -462,31 +375,13 @@ public void SetEnemyEquipment(int variant)
                 // they are better than the value in the definition.
                 for (int i = 0; i < ArmorValues.Length; i++)
                 {
+                     // BUG? this is using mobs armor val if it's less than equipment, contrary to comment above
                     if (ArmorValues[i] > (sbyte)(mobileEnemy.ArmorValue * 5))
```

Comment:
Possible bug here.

Reply:
I think this is because "no armor" = 100. Equipping armor subtracts from armor values, and unequipping adds it again (see DaggerfallEntity.UpdateEquippedArmorValues()).

A higher ArmorValues[i] means a lower actual armor rating, so in this case the code is working as commented by using the mobs armor rating if lower than equipment rating.

Reply:
Wow that's confusing but you're right. Backwards numbers confuse me :-)